docs/testing/tester-expected-macos.txt

```diff
@@ -0,0 +1,29 @@
++ cd ..
++ mkdir repo
++ cd repo
++ git init -q -b main
++ git config --local gpg.format ssh
++ git config --local user.signingkey ../keys/developer
++ gittuf trust init -k ../keys/root
+Signing data on standard input
+Signing data on standard input
++ gittuf trust add-policy-key -k ../keys/root --policy-key ../keys/policy.pem
+Signing data on standard input
+Signing data on standard input
++ gittuf policy init -k ../keys/policy --policy-name targets
+Signing data on standard input
+Signing data on standard input
++ gittuf policy add-rule -k ../keys/policy --rule-name protect-main --rule-pattern git:refs/heads/main --authorize-key ../keys/developer.pem
+Signing data on standard input
+Signing data on standard input
++ gittuf policy apply
+Signing data on standard input
++ echo 'Hello, world!'
++ git add .
++ git commit -S -m 'Initial commit'
```

Comment:
I think the single quotes being added on set -x is a bash thing. Maybe, if you explicitly run the script with bash, you could use the same expected output un ubuntu and macOS.